src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/visitbalances/VisitBalanceSynchronisationService.kt

```diff
@@ -25,7 +24,7 @@ class VisitBalanceSynchronisationService(
     val nomisPrisonNumber = event.offenderIdDisplay
     val telemetry = telemetryOf("visitBalanceAdjustmentId" to visitBalanceAdjustmentId, "nomisPrisonNumber" to nomisPrisonNumber)
 
-    if (event.doesOriginateInDps() && nomisApiService.isServicePrisonOnForPrisoner(serviceCode = VISIT_ALLOCATION_SERVICE, prisonNumber = nomisPrisonNumber)) {
+    if (event.originatesInDpsOrHasMissingAudit() && nomisApiService.isServicePrisonOnForPrisoner(serviceCode = VISIT_ALLOCATION_SERVICE, prisonNumber = nomisPrisonNumber)) {
```

Comment:
It is difficult to know how to handle missing audit so use DPS Allocation to determine most logical place.